api/test/integration/test_image_integration.py

```diff
@@ -0,0 +1,70 @@
+"""
+End-to-end API tests for images.
+
+Can be used to verify a live deployment is functioning as designed.
+Run with the `pytest -s` command from this directory.
```

Comment:
```suggestion
Run with the `pytest -s` command from this directory inside the deployed docker container.
```
I know this is from the old implementation, but I understood this wording as "this directory on my local machine", and it doesn't work. When I navigate into this directory in the local Docker container, the tests run.

Reply:
Makes sense, updated in 170226c.